app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -68,10 +68,11 @@
       android:layout_marginTop="8dp"
       android:cursorVisible="true"
       android:fontFamily="sans-serif"
-      android:inputType="@{viewModel.showPassword ? InputType.TYPE_CLASS_NUMBER : (InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_VARIATION_PASSWORD)}"
+      android:inputType="numberPassword"
       android:itemBackground="@color/white"
       android:textColor="@color/oppiaPrimaryText"
       android:textSize="14sp"
+      app:passwordHidden="@{viewModel.showPassword}"
```

Comment:
please correct the formatting. You can use reformat code from the menu.

Reply:
Also, check this setup for future work. https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros